conda-recipe/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set setup_py_data = load_setup_py_data(setup_file='../setup.py', from_recipe_dir=True) %}
+
+package:
+  name: ilastik-core
+  version: {{ setup_py_data.version }}
+
+source:
+  path: ..
+
+requirements:
+  host:
+    - python >=3.7,<3.10
+    - pip
+    - setuptools >=40.0
+    - setuptools_scm
+  run:
+    - python >=3.7,<3.10
+    # pin for numpy necessary because of vigra issue
+    # https://github.com/conda-forge/vigra-feedstock/pull/80#issuecomment-965105882
+    - numpy <1.20
```

Comment:
This pin is really unfortunate, as it will make this recipe incompatible with many existing environments. 
I see the test error in the conda build you linked to, but I have been using vigra with numpy>=1.20 for quite some time withouth any issues. For example this
```python
import numpy as np
import vigra
x = np.random.rand(64, 64).astype("float32")
y = vigra.filters.gaussianSmoothing(x, 1.0)
```
works in an env with vigra 1.11.1 and numpy 1.21.4 (both from conda-forge) and I have used many other "common" vigra functionality (mostly filters and segmentation functionality like watershed, connected components and relabeling) in that env without issues.

Have you ever run the relevant ilastik tests with a numpy>=1.20? I would argue that we don't need this pin if these tests pass, despite the failing conda tests. (Ofc it would be better to properly fix this, but I am not sure how easy that would be, esp. since vigra is pretty much unmaintained)

Reply:
yes, sure did. add a `x + y` to your code and you're there...

Reply:
version that will run into issues:

```python
import numpy as np
import vigra
x = vigra.taggedView(np.random.rand(64, 64).astype("float32"), axistags="xy")
y = vigra.filters.gaussianSmoothing(x, 1.0)
x + y

Reply:
This also fails:
```python
import numpy as np
import vigra
x = vigra.taggedView(np.random.rand(64, 64).astype("float32"), axistags="xy")
y = vigra.taggedView(np.random.rand(64, 64).astype("float32"), axistags="xy")
x + y
```
but this passes
```python
import numpy as np
import vigra
x = vigra.taggedView(np.random.rand(64, 64).astype("float32"), axistags="xy")
y = np.random.rand(64, 64).astype("float32")
x + y
print("Passed")
```
So the issue seems to be adding two vigra arrays with axistags (of course it was the axistags...).

> I think this is in practice only an issue with ilastikrag, so multicut.

Is this code that we could change to fix these issues? I think that this package will be much more useful if we don't need this numpy pin ...